### PR TITLE
Fix backend exit and cleanup

### PR DESCRIPTION
**Pull request by hexa00**
And also processManager binding see: #959

### CODE REVIEW

File: packages/core/src/node/backend-application.ts [hunk 1]
```diff
@@ -73,9 +73,9 @@ export class BackendApplication {
         // Handles normal process termination.
         process.on('exit', () => this.onStop());
         // Handles `Ctrl+C`.
-        process.on('SIGINT', () => this.onStop());
+        process.on('SIGINT', () => process.exit(0));
```
Comment (akosyakov) [inline packages/core/src/node/backend-application.ts]: What is the default if we remove it? The same question for `SIGTERM`.

**Comment by kittaakos** [comment]
One needs to verify two additional use-cases:
 - terminating the application that runs from source with `Ctrl+C` on Windows,
 - terminating a bundled electron application on Windows which was started from the terminal.

**Comment by kittaakos** [comment]
Please squash your changes. Thanks!

File: packages/core/src/node/backend-application.ts (hunk 1 above)
Comment (hexa00) [reply packages/core/src/node/backend-application.ts]: It exits with no exit event so no opportunity to call onstop

**Comment by hexa00** [comment]
@kittaakos these two issues are present without this PR right ? I did test that from source on windows and this PR doesn't fix that i found more info about it however see my recent comment on #633 

About squashing these commits fix a different problem at each commit so they are ok like that even if it's one line

File: packages/core/src/node/backend-application.ts (hunk 1 above)
Comment (akosyakov) [reply packages/core/src/node/backend-application.ts]: ok

**Comment by hexa00** [comment]
Oops I mean my comment on #815 but it should apply to the other too

**Comment by hexa00** [comment]
Note the appveyor build is a timeout in NodeFileSystem test

**Pull request merged**